src/main/java/suneido/runtime/builtin/Lucene.java

```diff
@@ -177,13 +181,14 @@ public static Object Search(Object self, Object a, Object b, Object c, Object d)
 			TopDocs results = searcher.search(query, limit);
 			SimpleHTMLFormatter htmlFormatter = new SimpleHTMLFormatter();
 			Highlighter highlighter = new Highlighter(htmlFormatter, new QueryScorer(query));
+			String key = "";
```

Comment:
Why did you move this out of the loop?

Reply:
Preventative measure. If there is nothing to loop through, key does not get initialized yet it is used following the loop

Reply:
I don't see it used outside the loop. Java wouldn't allow that.

Reply:
I'll double check and correct when I am back in tomorrow. Could have sworn it was passed into Ops.call1 but I must be mistaken